irc/handlers.go

```diff
@@ -804,6 +804,26 @@ func debugHandler(server *Server, client *Client, msg ircmsg.IrcMessage, rb *Res
 	case "STOPCPUPROFILE":
 		pprof.StopCPUProfile()
 		rb.Notice(fmt.Sprintf("CPU profiling stopped"))
+
+	case "CRASHSERVER":
+		if !client.HasRoleCapabs("oper:rehash") {
+			rb.Notice(client.t("You must have rehash permissions in order to execute DEBUG CRASHSERVER"))
+			return false
+		}
+		code := unregisterConfirmationCode(server.name, server.ctime)
```

Comment:
Might be nice if this is pulled out of chanserv.go and named something more generic?